app/controllers/declaration/AdditionalFiscalReferencesController.scala

```diff
@@ -111,6 +111,6 @@ class AdditionalFiscalReferencesController @Inject()(
     implicit request: JourneyRequest[AnyContent]
   ): Future[Option[ExportsDeclaration]] =
     updateExportsDeclarationSyncDirect { model =>
-      model.updatedItem(itemId, item => item.copy(additionalFiscalReferencesData = Some(updatedAdditionalFiscalReferencesData)))
+      model.updateItem(itemId, item => item.copy(additionalFiscalReferencesData = Some(updatedAdditionalFiscalReferencesData)))
```

Comment:
What is the difference between those methods?

Reply:
The letter 'd'

Reply:
I thought that there are two different method for update item, didn't notice that was rename

Reply:
It's been renamed back.  I thought it was a typo.  Apparently its named after scala convention for collection libraries.